app/templates/suppliers/companies_house_number.html

```diff
@@ -36,9 +37,7 @@
             value = form.companies_house_number.data,
             error = form.companies_house_number.errors[0],
             question_advice =
-            'This is a unique 8-character number given to your company by Companies House.
-            Find your <a href="https://beta.companieshouse.gov.uk/">Companies House number</a>.
-            ' | safe
+            '<a href="https://beta.companieshouse.gov.uk/">Find your Companies House number</a>' | safe
```

Comment:
Side note: this is what I see as one of the significant flaws of the frontend toolkit - I tend to use the `{% set ... %}` tag to set values like this.

Reply:
Auto escaping?  How's that?  I'm gonna try it out now.

Reply:
The same auto escaping all other regular text in jinja gets, no `|safe` required.

Reply:
or maybe a better way to put it... *without* escaping.

Reply:
Blimey wasn't even expecting it for that much.